clients/lsp-verilog.el

```diff
@@ -32,6 +32,117 @@
 
 (require 'lsp-mode)
 
+(defgroup lsp-svlangserver nil
+  "Settings for the SystemVerilog language server client."
+  :group 'lsp-mode
+  :tag "Language Server"
+  :link '(url-link "https://github.com/imc-trading/svlangserver")
+  :package-version '(lsp-mode . "7.1"))
+
+(defcustom lsp-clients-svlangserver-node-command "node"
+  "node binary path"
+  :group 'lsp-svlangserver
+  :type 'string
+  :safe (lambda (x) (stringp x)))
+
+(defcustom lsp-clients-svlangserver-module-path "svlangserver.js"
+  "svlangserver module path"
+  :group 'lsp-svlangserver
+  :type 'string
+  :safe (lambda (x) (stringp x)))
+
+(defcustom lsp-clients-svlangserver-workspace-additional-dirs nil
+  "Additional directories to be managed by this instance of svlangserver"
+  :group 'lsp-svlangserver
+  :type '(repeat string)
+  :safe (lambda (x) (seq-every-p #'stringp x)))
+
+(defcustom lsp-clients-svlangserver-includeIndexing '("**/*.{sv,svh}")
```

Comment:
list of string cannot be serialized with native jsons and it will fail with emacs 27.x. You have to replace all lists that are going to be used in lsp-register-custom-settings with vectors and use lsp-string-vector type.

Reply:
oh ok. I see the below in lsp-mode.el . So should I use lsp-repeatable-vector instead then?
```elisp
(define-widget 'lsp-repeatable-vector 'lsp--editable-vector
  "A variable length homogeneous vector."
  :tag "Repeat"
  :format "%{%t%}:\n%v%i\n")

(define-widget 'lsp-string-vector 'lazy
  "A vector of zero or more elements, every element of which is a string.
Appropriate for any language-specific `defcustom' that needs to
serialize as a JSON array of strings.

Deprecated. Use `lsp-repeatable-vector' instead. "
  :offset 4
  :tag "Vector"
  :type '(lsp-repeatable-vector string))

(make-obsolete 'lsp-string-vector nil "lsp-mode 7.1")
```